wscript

```diff
@@ -260,6 +260,12 @@ def build(bld):
         source = bld.path.find_node(lv2_h_path),
         target = bld.path.get_bld().make_node(lv2_h_path))
 
+    # Copy lv2_util.h to URI-style include path in build directory
+    lv2_util_h_path = 'lv2/lv2plug.in/ns/lv2core/lv2_util.h'
```

Comment:
Same as the code above it duplicates, this would be better as a for loop.

Could even be a glob to avoid this in the future, but explicit is probably safer.